yarn.lock

```diff
@@ -3973,6 +3982,11 @@ dotenv@^10.0.0:
   resolved "https://registry.yarnpkg.com/dotenv/-/dotenv-10.0.0.tgz#3d4227b8fb95f81096cdd2b66653fb2c7085ba81"
   integrity sha512-rlBi9d8jpv9Sf1klPjNfFAuWDjKLwTIJJ/VxtoTwIR6hnZxcEOQCZg2oIL3MWBYw5GpUDKOEnND7LXTbIpQ03Q==
 
+dotenv@^16.4.5:
```

Comment:
I'm surprised that you can add the dotenv package on the frontend, I swear I've tried but got some weird weird errors that only popup when trying to add that package on a react frontend to the point we felt it wasn't necessary

Reply:
how have you added env variables before?

Reply:
I remember it being this error [here](https://stackoverflow.com/questions/71019949/dotenv-issues-error-with-create-react-app-react-scripts) with react-scripts because of the create-react-app, I think it was just a basic yarn add dotenv, but if it works it works.

Reply:
removed dotenv since im using react env vars